pom.xml

```diff
@@ -88,7 +111,6 @@
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-slf4j-impl</artifactId>
-      <version>2.12.1</version>
```

Comment:
Why was version removed?

Reply:
Version control moved to dependencyManagement section https://github.com/folio-org/mod-finance/blob/c94932d0fb4bb22cd88288760bd34d0f1b6d12b6/pom.xml#L55